api/src/modules/h3-data/h3-data.entity.ts

```diff
@@ -7,7 +7,10 @@ import {
 } from 'typeorm';
 
 export interface H3IndexValueData {
-  [key: string]: number;
+  //H3 index
+  h: string;
```

Comment:
![triggered](https://media4.giphy.com/media/pYI1hSqUdcBiw/giphy.gif?cid=ecf05e47e4ikdybun6w8d7jyje7sl8a7fkzc2uzbs3w6m3zn&rid=giphy.gif&ct=g)

Reply:
- https://github.com/Vizzuality/landgriffon/pull/101/files#diff-46aa1ad8bbc9bdbf2afdf5de5ca7928aedf234a1881a252f83e6914c3d6b06dfR5

Sorry I though the interface definition comment was enough to clarify the purpose of the properties. A more extensive explanation available in the link above.

Thanks!

Reply:
Added **also** a comment to clarify the purspose of the interface and the reason behind the short prop names

Fix -> 0752f2ebaedcc5d72f18086f93ed6623ebc71e32